enhancements/storage/storage-hypershift.md

```diff
@@ -0,0 +1,1012 @@
+| manifests/0000_90_cluster-storage-operator_01_prometheusrbac.yaml     | N/A  | N/A   | RBAC for Prometheus to scan openshift-cluster-storage-operator namespace Something in the management cluster already gives Prometheus to scan hypershift namespace.                                  |
+| manifests/0000_90_cluster-storage-operator_02_servicemonitor.yaml     | CPO  | MC    | ServiceMonitor to scrape CSO metrics                                                                                                                                                                 |
+| manifests/02_csi_driver_operators_namespace.yaml                      | CVO  | G     | openshift-cluster-csi-drivers namespace                                                                                                                                                              |
+| manifests/03_credentials_request_*.yaml                               | N/A  | N/A   | CredentialRequests for all clouds. We use STS in AWS HyperShift, we assume CPO already provides cloud credentials.                                                                                   |
+| manifests/06_operator_cr.yaml                                         | CVO  | G     | operator CR                                                                                                                                                                                          |
+| manifests/07_serviceaccount.yaml                                      | CVO  | G     | Operator's ServiceAccount in the guest cluster                                                                                                                                                       |
+| NEW: CSO + AWSO ServiceAccounts                                       | CPO  | MC    | ServiceAccounts for the operator RBACs in the management cluster. Not sure if it's needed.                                                                                                           |
+| manifests/08_operator_rbac.yaml                                       | CVO  | N/A   | Operator RBAC in the guest cluster, basically cluster-admin binding.                                                                                                                                 |
+| manifests/09_metrics_service.yaml                                     | CPO  | MC    | Service for metrics                                                                                                                                                                                  |
+| manifests/10_deployment.yaml                                          | CPO  | MC    | Operator Deployment                                                                                                                                                                                  |
```

Comment:
same as above question about how we are going to keep these YAML synchronized?

Reply:
Answered above.